calculator/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
You put the `create-react-app` code into a subfolder, it would be nice to mention that in the PR instructions for developers trying to run your code.